packages/web-components/fast-foundation/package.json

```diff
@@ -25,7 +25,8 @@
     ".": {
       "types": "./dist/dts/index.d.ts",
       "default": "./dist/esm/index.js"
-    }
+    },
+    "./custom-elements": "./dist/custom-elements.json"
```

Comment:
I'm not sure if this is what you are asking exactly, but I was conflicted between calling it "manifest" or "custom-elements". In the end I settled with custom-elements because it matches the customElements field that is already present in the package.json.

Reply:
I can definitely update the path to custom-elements-manifest if we think the current change will be confusing to consumers.

Reply:
I’m open - would be good to hear from others (@nicholasrice, @EisenbergEffect perhaps). Approving regardless of the outcome here :)

Reply:
I like keeping it as `custom-elements` since that's what the other package property is named.